plugins/kubernetes/01general

```diff
@@ -16,11 +17,17 @@ SERVICES = ["etcdctl",
             "containerd",
             "dockerd"]
 
+ps_output = helpers.get_ps()
+if not ps_output:
+    print("not found k8s processes running, skipping")
+    sys.exit(0)
+
```

Comment:
We don't want to exit because it might still be possible to list pods/containers based in logs (see below). Also we want to always print "kubernetes:" so that we know the module ran. The idea i tried to use across plugins is that if there is no data it just prints empty (yaml) list.